js/api-augment/interfaces/augment-api-query.ts

```diff
@@ -292,6 +292,12 @@ declare module '@polkadot/api-base/types/storage' {
       [key: string]: QueryableStorageEntry<ApiType>;
     };
     msa: {
+     /**
+       * Storage type for key to MSA information
+       * - Key: AccountId
+       * - Value: [`MessageSourceId`]
+       **/
+      getMsaId: AugmentedQuery<ApiType, (arg: AccountId32 | string | Uint8Array) => Observable<Option<u64>>, [AccountId32]> & QueryableStorageEntry<ApiType, [AccountId32]>;
```

Comment:
Where is this coming from?

Reply:
I think this is a public function

Reply:
Yes this should not have been added

Reply:
@rlaferla  I think you also have to update metadeta fetch first and then build with chain running in instant sealing mode

Reply:
That's some stray code.  It's removed in the next commit.